modules/table/src/main/java/org/apache/ignite/internal/table/distributed/storage/InternalTableImpl.java

```diff
@@ -1997,7 +1997,7 @@ protected CompletableFuture<PendingTxPartitionEnlistment> enlist(int partId, Int
         Function<ReplicaMeta, PendingTxPartitionEnlistment> enlistClo = replicaMeta -> {
             ReplicationGroupId partGroupId = targetReplicationGroupId(partId);
 
-            tx.enlist(partGroupId, tableId, getClusterNode(replicaMeta), enlistmentConsistencyToken(replicaMeta));
+            tx.enlist(partGroupId, tableId, getClusterNode(replicaMeta).name(), enlistmentConsistencyToken(replicaMeta));
```

Comment:
Just use {{replicaMeta.getLeaseholder()}} here.

Reply:
done, thanks for suggestion